### DISCUSSION TITLE
wgpu crashes on Vulkan backend on Linux

### DISCUSSION THREAD

**Discussion by VarLad** [gfx-rs/wgpu#2376] [General]
Coming from https://github.com/iced-rs/iced/issues/1185 

Opening a discussion instead of an issue since I don't know if its a wgpu problem or iced problem

As given in the above issue, some examples in iced crash while resizing on X11 and don't even start on Wayland, giving the same same error in both cases

**Comment by kvark** [comment]
Please check wgpu's examples (including resizing) on your platform.

**Reply by VarLad** [reply]
@kvark All examples work perfectly, including resizing on both X11 and Wayland

**Reply by kvark** [reply]
Thank you! Then we need to figure out what iced is doing different.
